api_app/api/routes/airlock_resource_helpers.py

```diff
@@ -147,6 +148,7 @@ async def delete_review_user_resource(
         workspace_service_repo: WorkspaceServiceRepository,
         resource_template_repo: ResourceTemplateRepository,
         operations_repo: OperationRepository,
+        resource_history_repo: ResourceHistoryRepository,
```

Comment:
I strongly disagree.  Since the UI is using the history this PR will become huge and no way anyone can review it. Had this discussion with @tamirkamara  as well that this is going to be divided.

Reply:
Since the UI is using the history this PR will become huge and no way anyone can review it. Had this discussion with @tamirkamara as well that this is going to be divided.
Regarding the usage, it has all the details in the PR description